Examples/Swift/ViewController.swift

```diff
@@ -442,6 +452,20 @@ extension ViewController: NavigationViewControllerDelegate {
     }
 }
 
+// Mark: VisualInstructionDelegate
+extension ViewController: VisualInstructionDelegate {
+    func label(_ label: InstructionLabel, willPresent instruction: VisualInstruction, as presented: NSAttributedString) -> NSAttributedString? {
+        guard exampleMode == .customInstruction else { return nil }
```

Comment:
With `VoiceController(_:WillSpeak:Instruction:)`, returning nil omits the voice instruction. I'm guess that's not the case here? In reality, I can't really think of a use case where a developer would want to show nothing though.

Reply:
That's kind of the idea. The utility of "I only want to override default behavior under specific circumstances" seems to outweigh the utility of "it's nice to say 'show nothing' via `nil` instead of `NSAttributedString()`". In this _specific_ example you could _also_ implement default behavior by returning `presented`, but I think we should get into the general habit of using a  "returning nil means the delegate [has no answer](http://thecodelesscode.com/case/31) to the delegator, and thus default behavior should be used" pattern in our delegation, as otherwise you kind of force the developer to implement logic if they implement the optional delegate function. Not a super great pattern.